public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -87,6 +92,8 @@
     private Provider<CdrVersion> defaultCdrVersionProvider;
     @Autowired
     private ConceptService conceptService;
+    @Autowired
+    private GoogleAnalyticsServiceImpl googleAnalyticsService;
```

Comment:
Do you want the interface here, or does that not autowire correctly to the impl?

Reply:
It is the implementation being autowired. Bad named it with the interface name. Sorry. Corrected.